Documentation/k8s-monitoring.md

```diff
@@ -0,0 +1,50 @@
+# Monitoring
+Each Rook cluster has some built in metrics collectors/exporters for monitoring with [Prometheus](https://prometheus.io/).  To enable monitoring of Rook in your Kubernetes cluster, you can follow the steps below.
+Note that these steps work best with a local Kubernetes cluster running in `Vagrant`.
+
+## Prometheus Operator
+First the Prometheus operator needs to be started in the cluster so it can watch for our requests to start monitoring Rook and respond by deploying the correct Prometheus pods and configuration.
+A full explanation can be found in the [Prometheus operator repository on github](https://github.com/coreos/prometheus-operator), but the quick instructions can be found here:
+```
+pushd /tmp
+git clone https://github.com/coreos/prometheus-operator.git
+cd prometheus-operator
+kubectl apply -f deployment.yaml
+```
+This will start the Prometheus operator, but before moving on, wait until the operator is in the `Running` state:
+```
+kubectl get pod | grep prometheus-operator | awk '{print $3}'
+```
+Once the Prometheus operator is in the `Running` state, head back over to the Rook repo:
+```
+popd
+```
+
+## Prometheus Instances
+With the Prometheus operator running, we can create a service monitor that will watch the Rook cluster and collect metrics regularly.
+From the root of your locally cloned Rook repo, go the monitoring directory:
+```
+cd demo/kubernetes/monitoring
+```
+
+Create the service monitor as well as the Prometheus server pod and service:
+```
+kubectl create -f service-monitor.yaml 
+kubectl create -f prometheus.yaml
+kubectl create -f prometheus-service.yaml
+```
+
+Ensure that the Promtheus server pod is in the `Running` state before moving on:
+```
+kubectl -n rook get pod | grep  prometheus-rook | awk '{print $3}'
```

Comment:
can we remove the `awk` from these status checks? it would simplify the readability and it's pretty easy to pick out the status